test/fixtures/normalize-ways/12022001101.json

```diff
@@ -1,3 +1,3 @@
-{"type": "Feature", "properties": {"oneway": "1", "refs": ["1", "2", "3"], "id": "123|0!0;12022001101", "highway": "motorway", "openLR": "CwG/kSLC9wH1NvJS+WIBFA=="}, "geometry": {"type":"LineString", "coordinates": [[2.4585342407226562,48.883456897746534],[2.4451446533203125,48.877812415009195],[2.4235153198242188,48.866521538507754]]}}
-{"type": "Feature", "properties": {"oneway": "1", "refs": ["11", "22", "33"], "id": "223|0!0;12022001101", "highway": "motorway", "openLR": "CwG5MSK/4QHkNg2uBp4BBQ=="}, "geometry": {"type":"LineString", "coordinates": [[2.4235153198242188,48.866521538507754],[2.4451446533203125,48.877812415009195],[2.4585342407226562,48.883456897746534]]}}
-{"type": "Feature", "properties": {"oneway": "0", "refs": ["44", "55"], "id": "224|0!0;12022001101", "highway": "residential", "openLR": "CwG/kSLC9yj/Av/TAJcoHw=="}, "geometry": {"type":"LineString", "coordinates": [[2.4585342407226562,48.883456897746534],[2.4580836296081543,48.884966688891]]}}
+{"type": "Feature", "properties": {"oneway": 1, "refs": ["1", "2", "3"], "id": "123|0!0;12022001101", "highway": "motorway", "openLR": "CwG/kSLC9wH1NvJS+WIBFA=="}, "geometry": {"type":"LineString", "coordinates": [[2.4585342407226562,48.883456897746534],[2.4451446533203125,48.877812415009195],[2.4235153198242188,48.866521538507754]]}}
```

Comment:
why did you have to make these numbers? what is the usual input?

Reply:
the fixtures were actually invalid to begin with - graph-normalizer expects integer values here.